dts/bindings/timer/st,stm32-lptim.yaml

```diff
@@ -45,3 +45,10 @@ properties:
       - 32
       - 64
       - 128
+
+  st,counter-value:
+    type: int
+    description: |
+        Gives the LPTIM an exact counting value for timeout expiration.
```

Comment:
`        Gives the LPTIM an exact counting value (s) for timeout expiration.`

Reply:
line changed